ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -195,14 +211,16 @@ static void render_event(const char *event_name) {
   // Important note: We've observed some rendering issues in perfetto if the tid or pid are numbers that are "too big",
   // see https://github.com/ivoanjo/gvl-tracing/pull/4#issuecomment-1196463364 for an example.
 
+  if (strcmp(event_name, "started_tracing") != 0) {
```

Comment:
Minor: One alternative to avoid doing this every time would be to pass in the `previous_event_name` as NULL